src/common/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,69 @@
+import cn from 'classnames';
+import React, {Component} from 'react';
+import ITabsProps from './Tabs.d';
+
+import styles from './tabs.scss';
+
+export default class Tabs extends Component<ITabsProps> {
+	public static defaultProps = {
+		tabs: [
+			{
+				title: '1',
+				content: 'content',
+			},
+		],
+		tabsHeight: 50,
+	};
+
+	public state = {
+		currentTab: 0,
+	};
+
+	public tabStyle = (index: number) => {
+		const {currentTab} = this.state;
+		const {tabsHeight, style} = this.props;
+		const idleStyle = style ? style.tab : undefined;
+		const activeStyle = style ? style.activeTab : undefined;
+
+		const baseStyle = index === currentTab ? activeStyle : idleStyle;
+
+		return {
+			...baseStyle,
+			height: tabsHeight,
+		};
+	}
+
+	private readonly onTabChange: any = (index: number) => {
+		this.setState({currentTab: index});
+	}
+
+	public render() {
+		const {tabs, tabsHeight} = this.props;
+		const {currentTab} = this.state;
+
+		return (
+			<div className={styles.tabs}>
+				<div className={styles.tabsList}>
+					{
+						tabs.map((tab, index) => (
+							<div
+								key={index}
+								className={cn(
+									styles.tab, {
+									[styles.tab_active]: index === currentTab,
+								})}
+								style={this.tabStyle(index)}
+								onClick={() => this.onTabChange(index)}
+							>
+								<span className={styles.tab_title}>{tab.title}</span>
+							</div>
+						))
+					}
+				</div>
+				<div style={{top: tabsHeight}} className={styles.tabContent}>
```

Comment:
why top? why tabsHeight? use a consistent naming.

Also I think you must use a default position, because tabs component woudn't be a absolutely positioned